guildwars2/characters.py

```diff
@@ -459,3 +683,43 @@ def get_crafting(self, character):
             discipline = crafting["discipline"]
             craft_list.append("Level {} {}".format(rating, discipline))
         return craft_list
+
+    def search_lvl_to_increase(self, level: int, lvl_dict):
+        for increase, lvl in lvl_dict.items():
+            if lvl[0] <= level <= lvl[1]:
+                if level < 11:
+                    return increase
+                elif level % 2 == 0:
+                    return increase
+                else:
+                    return 0
+
+    def calcBaselvl(self, level: int, acc_baselvl: int, lvl_dict):
```

Comment:
Style: use snake_case for function names

Reply:
Refactored calc_base_health() and calc_base_lvl()